linux/open_vm_tools/add_enable_service.yml

```diff
@@ -56,11 +56,19 @@
         service_enabled: true
         service_state: "started"
 
-- name: "Enable FreeBSD service {{ service_name }}"
+- name: "Update tools.conf path and enable FreeBSD service {{ service_name }}"
   when:
     - guest_os_family == 'FreeBSD'
     - service_name == 'vmware-guestd'
   block:
+    - name: "Update tools.conf path to /etc/vmware-tools/tools.conf when tools is installed from source code"
+      ansible.builtin.replace:
+        path: "{{ service_file }}"
+        regexp: '/usr/local/share/vmware-tools/tools.conf'
+        replace: '/etc/vmware-tools/tools.conf'
+      delegate_to: "{{ vm_guest_ip }}"
+      when: linux_ovt_install_type == 'source'
```

Comment:
Only OVT source install will execute tasks in this file. So here doesn't need to check `linux_ovt_install_type`. But you need to check `service_file_local_path` because only when the service file is copied from an original one, the tools.conf path need to be updated.
```
    - service_file_local_path is defined
    - service_file_local_path
 ```

For service file created from the template, it has correct tools.conf path as you updated